cypress/support/commands/byTbp.js

```diff
@@ -0,0 +1,79 @@
+/*
+@kuba idk where all these should be placed so i just threw them here.
+
+Pls tell me the _organisation_ standards for cypress/js :^).
+ */
+
+
+import {getLanguageSpecific} from "../utils/utils";
+
+Cypress.Commands.add('getCurrentRecordId', () => {
+  describe('Select the current record ID from the url', function () {
+    let currentRecordId = 0;
+    cy.url().then(ulrr => {
+      currentRecordId = ulrr.split('/').pop();
+    });
+    return currentRecordId;
+  });
+});
+
+
+Cypress.Commands.add('setCheckBoxValue', (fieldName, isChecked, modal = false, rewriteUrl = null) => {
```

Comment:
i believe this one should go to `form.js`

Reply:
why is `clickOnCheckBox` not enough ?

Reply:
done